components/Select.tsx

```
@@ -111,11 +112,34 @@ const Select = forwardRef<HTMLDivElement, SelectProps>(
+    useEffect(() => {
+      const filteredOptions = options.filter((option) =>
+        getOptionLabel(option).toLowerCase().includes(searchValue.toLowerCase())
+      );
+
+      onOptionsCountChange?.(filteredOptions.length);
+    }, [options, searchValue, onOptionsCountChange]);
+
+    useEffect(() => {
+      const handleWheel = (e: WheelEvent) => {
+        const optionsElement = selectRef.current?.querySelector(".options");
+        if (optionsElement && optionsElement.contains(e.target as Node)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)